packages/gestalt-codemods/0.51.0-0.52.0/change-default-box-display.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Converts
+ *  <Box display="flex" />
+ * to
+ *  <Box />
+ *
+ * Converts
+ *  <Box />
+ * to
+ *  <Box display="block" />
+ */
+export default function transformer(file, api) {
```

Comment:
Would love to know the results of running this internally - is it totally crazy?

Reply:
@chrislloyd fairly crazy 😊 
Running against `webapp/app/`:
```
2738 errors
2629 unmodified
3 skipped
115 ok
```
All errors appear to be the same `SyntaxError: Unexpected token, expected ","`. Have you seen that before?

Reply:
No - I'd dig into that. It's probably a sign that something is wrong with your script or you're running it with an incorrect Node version.